src/blog/2020-04-01/stryker-version-3.pug

```diff
@@ -0,0 +1,41 @@
+extends ../../layout/blog-layout.pug
+block content
+    :markdown-it
+        Over the years, we've been privileged to explain the concept of Mutation Testing to great developers, testers and others. 
+        The concept of _insert bugs your production code_ in order to _test your tests_ sparks the imagination. 
+
+        But when people start using mutation testing in practice, they 
+        quickly realize that things are not as easy as they seem. 
+        One of the most common feedback is: what 'mutation score' is good enough? 
+        Our general advice is not to strive for 100%, but should you use 80%? Or something else? 
+        Some users just want to see that everything is great with their tests.
+
+        That's why we're introducing a new '100%' mode!
+        With this mode, Stryker itself will strive to improve your mutation score to be 100%. 
+        You can activate it with the `--100` command line flag, 
+        but we're thinking of making this the default mode.
+
+        ## 🙉 An example
+
+        A picture says more than a thousand words. First without the new `--100` mode.
+
+    img(alt="Without 100 mode", src="/images/blogs/stryker-without-100-mode.png" class="img-fluid")
+
+    :markdown-it
+        And now with the new `--100` mode.
+
+    img(alt="With 100 mode", src="/images/blogs/stryker-with-100-mode.png" class="img-fluid")
+
+    :markdown-it
+        As you can see, this new mode makes for a far better-looking report! And things just getting better, since all mutants will be _killed, no matter what_, 
```

Comment:
```suggestion
        As you can see, this new mode makes for a far better-looking report! And things are just getting better, since all mutants will be _killed - no matter what_.
```